tasks/system/Linux.yml

```diff
@@ -69,24 +67,27 @@
   become: True
   when: not solr_service_status
 
-- name: "Unarchive and install solr {{ solr_version }}"
+- name: 'Unarchive solr {{ solr_version }}'
   unarchive:
-    creates: "{{ dest_solr_path }}"
-    src: "{{ dest_solr_path }}.zip"
-    dest: "{{ dest_main_path }}"
+    creates: '{{ dest_solr_path }}'
+    src: '{{ dest_solr_path }}.zip'
+    dest: '{{ dest_main_path }}'
+    owner: '{{ solr_user }}'
+    group: '{{ solr_group }}'
     remote_src: True
   become: True
   when: not solr_service_status
 
-- name: "Install solr as service"
-  command: /bin/bash {{ dest_solr_path }}/bin/install_solr_service.sh \
+- name: Install solr as service
+  command: >-
+    /bin/bash {{ dest_solr_path }}/bin/install_solr_service.sh
```

Comment:
We really need bash here?

Reply:
We r using the default functionality to setup service

Reply:
will use systemd to setup service